src/main/java/it/pagopa/afm/marketplacebe/service/BundleOfferService.java

```diff
@@ -85,6 +75,13 @@ public List<BundleOffered> sendBundleOffer(String idPsp, String idBundle, CiFisc
 
         List<BundleOffered> bundleOfferedList = new ArrayList<>();
         ciFiscalCodeList.getCiFiscalCodeList().forEach(fiscalCode -> {
+            // check if the bundle has already been offered
+            var ciBundle = ciBundleRepository.findByIdBundleAndCiFiscalCode(idBundle, fiscalCode);
+            if (ciBundle.isPresent()) {
```

Comment:
could a bundle offer be send to all or nobody? 
how do you think it?

reading the code the behaviour seems be ambiguous

Reply:
fixed